images/compose/compose.folders.yml

```diff
@@ -1,30 +1,46 @@
-version: '3'
+# This file is intended as a starting point. Before using it, make a copy and adjust at least the passwords.
```

Comment:
Shouldn't this comment make reference to the new password files? It kind of reads like the passwords should be changed here.